frontend/packages/ceph-storage-plugin/src/components/bucket-class/wizard-pages/general-page.tsx

```diff
@@ -9,23 +9,15 @@ import {
   TextInput,
   Radio,
   ValidatedOptions,
-  Tooltip,
 } from '@patternfly/react-core';
-import { ExternalLink } from '@console/internal/components/utils';
-import { OutlinedQuestionCircleIcon } from '@patternfly/react-icons';
+import { ExternalLink, FieldLevelHelp } from '@console/internal/components/utils';
```

Comment:
I can see all three files have, three different ways of importing FieldLevelHelp. better you choose one import

Reply:
better use
import { FieldLevelHelp } from '@console/internal/components/utils/field-level-help';